config/application.yml.default

```diff
@@ -130,6 +130,7 @@ in_person_email_reminder_early_benchmark_in_days: 11
 in_person_email_reminder_final_benchmark_in_days: 1
 in_person_email_reminder_late_benchmark_in_days: 4
 in_person_proofing_enabled: false
+in_person_proofing_opt_in_option: false
```

Comment:
Slightly different naming to indicate it's a boolean and to be consistent with other feature flag names
```suggestion
in_person_proofing_opt_in_enabled: false
```